Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.yml

```diff
@@ -0,0 +1,157 @@
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- defaultvalue: '20160'
+  display: ''
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: '240'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- additionalinfo: Type of the indicator in the feed. If auto-detect is checked then the value set as Indicator Type will be ignored.
+  display: Indicator Type
+  name: indicator_type
+  required: false
+  type: 0
+- additionalinfo: Free text actor search (all fields included).
+  display: Actor
+  name: actor
+  required: false
+  type: 0
+- additionalinfo: How far back in time to go when performing the first fetch
+  defaultvalue: '7 days'
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  hidden: false
+  name: fetch_time
+  required: false
+  type: 0
+- additionalinfo: 'Long unix time. Search indicators with activity range overlapping given time (including). Object fields: activity.first and activity.last. Empty indicates unbounded'
+  display: Start Time
+  name: from
+  required: false
+  type: 0
+- additionalinfo: 'Long unix time. Search indicators with activity range overlapping given time (including). Object fields: activity.first and activity.last. Empty indicates unbounded'
+  display: End Time
+  name: until
+  required: false
+  type: 0
+- display: Username
+  name: credentials
+  required: false
+  type: 9
+- additionalinfo: JMESPath expression for extracting the indicators. You can use http://jmespath.org/
+    to identify the proper expression.
+  display: JMESPath Extractor
+  name: extractor
+  required: true
+  type: 0
+- additionalinfo: The JSON attribute that holds the indicator value. Default value
+    is 'indicator'.
+  display: JSON Indicator Attribute
+  name: indicator
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- additionalinfo: Supports CSV values.
+  display: Tags
+  hidden: false
+  name: feedTags
+  required: false
+  type: 0
+description: Fetches indicators from a JSON feed.
+display: Intel471 Actors Feed
+name: Intel471 Actors Feed
+script:
+  commands:
+  - arguments:
+    - default: false
+      defaultValue: '50'
+      description: The maximum number of results to return. The default value is 50.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets the feed indicators.
+    execution: false
+    name: intel471-get-indicators
```

Comment:
we do not want the same cmd to be in 2 different integrations. please rename to `intel471(-)actors-get-indicators`